lib/SIL/SILBasicBlock.cpp

```diff
@@ -355,3 +355,7 @@ bool SILBasicBlock::isTrampoline() const {
     return false;
   return begin() == Branch->getIterator();
 }
+
+bool SILBasicBlock::isLegalToHoistInto() const {
+  return true;
```

Comment:
Jordan pointed out via IM that this is used in other parts of loop info that we may use and that the old behavior was to always hoist. So given that, I am fine just taking this if it gets the build going. We can fix the test failures later. But at least we know things build.